doc/reference/causality.rst

```diff
@@ -0,0 +1,5 @@
+==========================
+Granger causality measures
```

Comment:
Is this meant only for the granger functions (as the title suggests), or all causality functions (as the file name suggests)? I think it should read "Causality", which would also sort it correctly in the function overview of the docs.

Reply:
It is meant for all of the causality measures so in that case it should be 'Causality'. However, we could also consider reorganisation into [functional connectivity tree](https://www.frontiersin.org/files/Articles/165147/fnsys-09-00175-HTML/image_m/fnsys-09-00175-g001.jpg). Full paper [here](https://www.frontiersin.org/articles/10.3389/fnsys.2015.00175/full).